types/tape/OTHERFILES.txt

```diff
@@ -0,0 +1 @@
+lib/test.d.ts
```

Comment:
this should be `lib/tape.d.ts` to match the filename (or maybe the filename should change to be test.d.ts)